tools/bench_connect.c

```
@@ -83,87 +83,132 @@ static void *thread_worker(void *varg) {
+    fprintf(stderr, "\"threads\":%d,", threads);
+    fprintf(stderr, "\"iterations\":%d,", n);
+    fprintf(stderr, "\"errors\":%d,", errors);
+    fprintf(stderr, "\"connections_per_sec\":%.1f,", cps);
+    fprintf(stderr, "\"connect_us\":{\"min\":%.1f,\"avg\":%.1f,\"p50\":%.1f,\"p90\":%.1f,\"p99\":%.1f,\"max\":%.1f},",
+            sorted_conn[0], avg_conn,
+            percentile(sorted_conn, total_n, 0.50),
+            percentile(sorted_conn, total_n, 0.90),
+            percentile(sorted_conn, total_n, 0.99),
+            sorted_conn[total_n-1]);
+    fprintf(stderr, "\"close_us\":{\"min\":%.1f,\"avg\":%.1f,\"p50\":%.1f},",
+            close_us[0], avg_close, close_us[total_n/2]);
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)